index.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 import { NativeModules, Platform } from 'react-native';
+import {PromptInfoOptions} from "react-native-keychain"
```

Comment:
Hi @jenskuhrjorgensen , please check it, why do we import `PromptInfoOptions` from  `react-native-keychain`

Reply:
I have no idea. Fixed :)